comps/third_parties/vllm/src/build_docker_rocm_vllm.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
You don't need such scripts here, those kind of scripts are used for images build from 3rd party dockerfile, like https://github.com/opea-project/GenAIComps/blob/9d9cbb61c1e552387dd00188e7349a5aaf8be3f8/comps/third_parties/vllm/src/build_docker_vllm.sh#L37